package.json

```diff
@@ -27,31 +27,29 @@
     "ember-blog-engine": "git://github.com/srowhani/ember-blog-engine.git#caf6e66089b9639cbb076b5eeaa656161f91a5b9",
     "ember-browserify": "~1.1.0",
     "ember-cli": "~2.8.0",
-    "ember-cli-app-version": "^1.0.0",
     "ember-cli-code-coverage": "0.3.5",
     "ember-cli-dependency-checker": "^1.2.0",
     "ember-cli-deploy": "~0.6.4",
     "ember-cli-deploy-frost-pack": "git://github.com/ciena-frost/ember-cli-deploy-frost-pack.git#e98e53709a42604815348f10bc8823c3c425b68e",
     "ember-cli-htmlbars-inline-precompile": "^0.3.1",
     "ember-cli-inject-live-reload": "^1.4.0",
-    "ember-cli-mocha": "^0.10.1",
+    "ember-cli-mocha": "^0.13.0",
```

Comment:
probably missing the new chai addon. `ember g ember-cli-mocha` to fix it up. 
also switch the resulting packages to `~` since they are <1.0.0